CIP-????/README.md

```diff
@@ -0,0 +1,311 @@
+  - The **left**/**right** position within a node follows the **`even-odd`** rule:  if its index `i` is even, the child is on the left side; otherwise, it's on the right. This position is determined by **`Pn = In - 2Ip`** (`Ip` is the parent's index)  where `Pn` is 0 when the child is on the left, 1 when it's on the right, and results in an **error** otherwise.
+- In order to keep track of changes in the state of the `T` tree, there are two types of nodes:
+  - Updateable node (**`Nu`**): These nodes are already part of the tree, but their values will be updated sometime by an update process.
+  - Parent node (**`Np`**): These are then nodes where a new leaf `L` will be appended to or the last leaf removed from.
+- The right child (`Nr`) of an parent node (`Np`), must always be an empty hash, represented as **`ε`**. The left child can be either an empty hash (**`ε`**) or a leaf.
+
+## Rationale: how does this CIP achieve its goals?
+
+The solution described in this CIP achieves the goal of ensuring the uniqueness of NFTs minted on the Cardano blockchain by using an off-chain data structure to track the minting and burning of NFTs, and storing the hash of this data structure together with the proofs on the blockchain. This allows the on-chain code to verify the uniqueness of an NFT at minting time, without requiring the entire data structure to be stored on the blockchain.
+
+Also, it allows anyone to independently build the off-chain data tree by retrieving on-chain minting/burning transactions. Therefore, there is no need for centralized solutions to validate the integrity of the tree, promoting decentralization and transparency within the Cardano ecosystem. Though, any - even un-trusted or adversarial -  3rd-party services can be used to be queried for a proof.
+
+The use of IntegriTree as the off-chain data structure allows for efficient and secure tracking of the minting and burning of NFTs. IntegriTree is a new accumulator scheme that leverages a complete binary hash tree for compact accumulation and dynamic updates. 
+
+The tree is used to accumulate the elements in the set, and to provide proofs. 
+The values are represented as open-interval pairs of consecutive elements with each node in the tree. IntegriTree offers proofs for membership, non-membership, addition, and deletion, with a proof size that scales logarithmically with the number of elements.
+
+The use of an accumulator to represent the state of the IntegriTree allows for the updated state to be verified and recorded on the blockchain in a single transaction. This simplifies the process of updating the state, and reduces the risk of errors or disputes.
+
+
+## Path to Active
+
+### Acceptance Criteria
+The acceptance criteria for this CIP to become active are as follows:
+
+- The solution described in the CIP has been implemented and tested on the Cardano blockchain.
+- The implementation has been reviewed and approved by the Cardano Foundation and IOHK.
```

Comment:
```suggestion
- The implementation has been reviewed and approved by subject matter experts.
```
Same rationale as the comment below r.e. not requiring a participant from a particular organization but instead someone whose personal credentials make them an expert worthy of providing peer review to the subject at hand.